frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx

```diff
@@ -236,12 +237,27 @@ class Visualization extends PureComponent {
     const question = this._getQuestionForCardCached(metadata, card);
     const mode = this.getMode(this.props.mode, question);
 
-    return mode
```

Comment:
should we leave this namespace out for now?

Reply:
Probably yes. It was for my own testing.